### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by alizaamir123**
When the user submits the form, you check if the email is in lower case.
If the validation is OK, the form is sent.
If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.
this project is completed  by @alizaamir123  and @muneeb3123

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @alizaamir123, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the student's success team and they'll permit you to work solo or look for a suitable partner.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by muneeb3123** [comment]
> 

hello brother we already did pair programming  kindly check

**Comment by topeogunleye** [comment]
> > 
> 
> hello brother we already did pair programming kindly check

@alizaamir123, Kindly submit for another code reviewer to review pls. I thought it was the same person because of Your similar profile pictures.

**Comment by muneeb3123** [comment]
> > > 
> > 
> > 
> > hello brother we already did pair programming kindly check
> 
> @alizaamir123, Kindly submit for another code reviewer to review pls. I thought it was the same person because of Your similar profile pictures.

OK OK no problem brother

File: Js/main.js [hunk 1]
```diff
@@ -11,11 +17,31 @@ function toggleMenue() {
...
+});
+submit.forEach((button) => {
+  button.addEventListener('click', togglePopup);
+});
+hidePopup.addEventListener('click', togglePopup);
+form.addEventListener('submit', (e) => {
+  if (email.value.toLowerCase() !== email.value) {
+    e.preventDefault();
+    errorMsg.style.display = 'block';
+    errorMsg.innerHTML = 'Email must be in Lowercase';
```
Comment (Zuhaib042) [inline Js/main.js:43]: - [ ] Good job on showing the message, but the problem is that the message is getting shown at top of the contact section. even on my first tries, I got confused and could not see the message shown. so this is not a good user experience. kindly consider showing this message around the input fields or around submit button. you dont need to add the background color, and margin, paddings in this div. you can simply show a text message in red color if not having extra space around input or submit is a issue.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @alizaamir123 @muneeb3123 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Implement most of the requirements


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @alizaamir123 @muneeb3123 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Implement most of the requirements


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @alizaamir123 @muneeb3123 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Implement most of the requirements


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — APPROVED** [review]
Hi @alizaamir123 and @muneeb3123, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Form validations are working ✔️ 
- Error message shows near input 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> > > > 
> > > 
> > > 
> > > hello brother we already did pair programming kindly check
> > 
> > 
> > @alizaamir123, Kindly submit for another code reviewer to review pls. I thought it was the same person because of Your similar profile pictures.
> 
> OK OK no problem brother

Apologies bro for the confusion. Thanks for understanding

**Pull request merged**